rust/processor/src/processors/token_v2_processor.rs

```diff
@@ -354,6 +354,7 @@ fn insert_current_collections_v2_query(
                 table_handle_v1.eq(excluded(table_handle_v1)),
                 token_standard.eq(excluded(token_standard)),
                 last_transaction_version.eq(excluded(last_transaction_version)),
+                collection_properties.eq(excluded(collection_properties)),
```

Comment:
You also need to update insert_collections_v2_query so that it backfills the new column

Reply:
updated!